internal/services/signalr/web_pubsub_hub_resource.go

```diff
@@ -105,6 +106,46 @@ func resourceWebPubSubHub() *pluginsdk.Resource {
 				},
 			},
 
+			"event_listener": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"user_event_name_filter": {
+							Type:     pluginsdk.TypeSet,
```

Comment:
This probably wants to be a list instead of a set 
```suggestion
							Type:     pluginsdk.TypeList,
```